scheduler/src/cook/compute_cluster.clj

```diff
@@ -530,11 +530,12 @@
         dissoc-ca-cert-vals
         (fn [configs]
           (map-vals dissoc-ca-cert configs))]
-    (log/info "Updating dynamic clusters."
-              {:current-configs (dissoc-ca-cert-vals current-configs)
-               :force? force?
-               :new-configs (dissoc-ca-cert-vals new-configs)
-               :updates (map #(update % :goal-config dissoc-ca-cert) updates)})
+    (doseq [current-config (dissoc-ca-cert-vals current-configs)]
```

Comment:
This can be a destructuring:

`(doseq ([cluster-name config] (dissoc-ca-cert-vals ...)))
   ... `

See: https://clojure.org/guides/destructuring

(Thats why you find relatively few `(key ...)` or `(val ...)` or direct dictionary gets in clojure

Also I suggest reformatting these outputs to make it easier to distinguish the in elk: 

`(log/info "Updating dynamic cluster" name "current config" ... )`

vs

`(log/info "Updating dynamic cluster current config for" name "to" ...)`

I can distinguish these much easier with the renamed version `"Updating dynamic cluster current config for"` more than I can `"Updating dynamic cluster"` Also, punctuation doesn't index will in lucene.